lms/static/certificates/sass/_components.scss

```diff
@@ -510,10 +507,6 @@
         .deco-corner-br {
             @include triangle(20px, $cert-distinguished-color, down-left);
         }
-
-        .organization-logo {
-            @include filter(grayscale(0%));
```

Comment:
Not needed any more due to https://github.com/edx/edx-platform/pull/8319/files#r31760171